packages/blocks/src/api/parser/get-block-attributes.js

```diff
@@ -58,6 +68,9 @@ export const toBooleanAttributeMatcher = ( matcher ) =>
  */
 export function isOfType( value, type ) {
 	switch ( type ) {
+		case 'rich-text':
+			return value instanceof RichTextData;
```

Comment:
is there any reason to _also_ consider a string value a valid `rich-text` value which we can then auto-convert when loading?

Reply:
I meant specifically in this line of code. my thinking was that some backend might send a string value here after parse and it's not yet converted to `RichText` but it's still _a rich-text attribute_

Reply:
At which stage can they do that?

Reply:
supposing the backend stores the raw JSON values representing the blocks and their attributes, it seems like they would dump that into the editor directly. maybe they have (or more likely, _had_) strings from an earlier version of the editor and now the new version returns the structure, but we start out with text.

Reply:
I think this check here is specifically for the parser result though?

Reply:
if that's the case it's fine. I only wanted to make sure we think about that situation if it applies, since it seemed like it _could_ be a thing we overlook.